lib/hal/debug.c

```diff
@@ -63,6 +63,10 @@ static void drawChar(unsigned char c, int x, int y, int fgColour, int bgColour)
 					*((short*)videoBuffer) = colourToDraw & 0xFFFF;
 					videoBuffer += sizeof(short);
 					break;
+				case 15:
+					*((short*)videoBuffer) = colourToDraw & 0xFFFF;
```

Comment:
Same exactly as case for 16, prefer merging and having two case labels if we handle them exactly the same

Reply:
Changed; Only kept the case label